custom_components/healthchecksio/sensor.py

```
@@ -0,0 +1,95 @@
+            else:
+                self._attr_icon = ICON_DEFAULT
+            self._attr_native_value = self._attr_native_value.title()
+        else:
+            self._attr_icon = ICON_DEFAULT
+        self._attr_extra_state_attributes[ATTR_ATTRIBUTION] = ATTRIBUTION
+        self._attr_extra_state_attributes[ATTR_LAST_PING] = self.check.get(
+            ATTR_LAST_PING
+        )
+
+    @property
+    def device_info(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `device_info`

(ANN201)